spring-data-relational/src/main/java/org/springframework/data/relational/core/dialect/PostgresDialect.java

```diff
@@ -214,8 +214,19 @@ public IdentifierProcessing getIdentifierProcessing() {
 	 */
 	@Override
 	public Set<Class<?>> simpleTypes() {
+
 		Set<Class<?>> simpleTypes = new HashSet<>();
-		ifClassPresent("org.postgresql.util.PGobject", simpleTypes::add);
+		final List<String> simpleTypeNames = Arrays.asList( //
```

Comment:
Nit: `final`

Reply:
Aaaaargh, I have disabled that in IntelliJ, but it must have a second setting for it somewhere!